### DISCUSSION TITLE
Saving with :w adds a new line

### DISCUSSION THREAD

**Discussion by vddo** [helix-editor/helix#10345] [General]
### Summary

I was playing with the lichess.org python client - berserk and got error messages regarding faulty HTTP requests. 

Turned out that helix adds a new line to my token-file. And because I only read the whole file, assuming its a one-liner, the '/n' character was being detected by the python library. 

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Linux - Ubuntu 22

### Terminal Emulator

GNOME Terminal Version 3.44.0 for GNOME 42

### Installation Method

dnf install helix

### Helix Version

helix 23.10 (f6021dd0)

**Comment by pascalkuthe** [comment]
This is intentional there is a setting tp turn this off
